scripts/populate-git-env.sh

```diff
@@ -0,0 +1,7 @@
+# List of environment variables you want to have access during next development and deployment.
+# To have access to your variables (e.g: echo $GIT_COMMIT_SHA) you have to use "source" like this: `source ./scripts/populate-git-env.sh`
```

Comment:
Wouldn't it be better to have a script for this? like "script:populate-git-env:print" or similar? Usually easier to find.